README.md

```diff
@@ -6,11 +6,21 @@
 [![Platform](https://img.shields.io/cocoapods/p/SettingsKit.svg?style=flat)](http://cocoapods.org/pods/SettingsKit)
 [![Sponsored by Detroit Labs](https://img.shields.io/badge/sponsor-Detroit%20Labs-000000.svg?style=flat)](http://www.detroitlabs.com)
 
-`SettingsKit` is a small library & build tool simplifies reading & writing items
-in the iOS Settings app. It also makes working with app settings a bit safer by
-using enums instead of magic strings that could accidentally be typo-ed.
+`SettingsKit` is a small library & build tool that makes it easier to work
+with app preferences in the iOS Settings app. It also makes working with
+settings a bit safer by using enums instead of "magic" strings, which are
+vulnerable to typos.
 
-![Settings Example](Docs/Screenshots/settings.png)
+
+## How does it work?
+`SettingsKit` comes bundled with a tool that generates an enum (`Settings.swift`)
```

Comment:
It's a bit odd to mention the filename here - only the enum name is important to the consumer, so maybe tweak to

> `SettingsKit` comes bundled with a tool that automatically generates a `Settings` enum...